controllers/ibmpowervscluster_controller_test.go

```diff
@@ -839,6 +869,454 @@ func TestIBMPowerVSClusterReconciler_delete(t *testing.T) {
 			g.Expect(clusterScope.Client.Update(ctx, powervsImage2)).To(Not(Succeed()))
 		})
 	})
+
+	t.Run("when delete TransitGateway returns error", func(t *testing.T) {
+		g := NewWithT(t)
+		clusterScope = powervsClusterScope()
+		clusterScope.IBMPowerVSCluster.Status = infrav1beta2.IBMPowerVSClusterStatus{
+			ServiceInstance: &infrav1beta2.ResourceReference{
+				ID: ptr.To("serviceInstanceID"),
+			},
```

Comment:
can we move this to the common function `powervsClusterScope` as this is needed for all test cases?

Reply:
Done.